packages/encoding/src/hex.ts

```diff
@@ -1,9 +1,11 @@
-export function toHex(data: Uint8Array): string {
-  let out = "";
-  for (const byte of data) {
-    out += ("0" + byte.toString(16)).slice(-2);
+function* map<T, U>(iterable: Iterable<T>, fn: (value: T) => U): IterableIterator<U> {
```

Comment:
```suggestion
// Helper function needed to support environments that do not yet have
// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Iterator/map
function* map<T, U>(iterable: Iterable<T>, fn: (value: T) => U): IterableIterator<U> {
```

@dynst is this correct?